spring-integration-sftp/src/test/java/org/springframework/integration/sftp/config/InboundChannelAdapterParserTests-context.xml

```diff
@@ -42,6 +42,7 @@
 								  temporary-file-suffix=".bar"
 								  comparator="comparator"
 								  local-filter="acceptAllFilter"
+								  max-fetch-size="42"
```

Comment:
Yeah - I thought about that (deprecating and standardizing) but when I looked at the JDBC adapter it's not nice - it fetches and discards rather than limiting the fetch.

Also, with JDBC each vendor has a mechanism in the query .... e.g. `... LIMIT 1`.